auto_submit/lib/service/github_service.dart

```diff
@@ -47,14 +47,22 @@ class GithubService {
   }
 
   /// Create a new issue in github.
-  Future<Issue> createIssue(
-    RepositorySlug repositorySlug,
-    String title,
-    String body,
-  ) async {
+  Future<Issue> createIssue({
+    required RepositorySlug repositorySlug,
```

Comment:
nit: general nomenclature is to simplify it and call it "slug"

Reply:
Okay no problem. I'll change it.